common_awsim_labs_sensor_launch/config/filter.param.yaml

```diff
@@ -0,0 +1,8 @@
+/**:
+  ros__parameters:
+    input_frame: ""
+    output_frame: ""
+    max_queue_size: 5
+    use_indices: false
+    latched_indices: false
+    approximate_sync: false
```

Comment:
@vividf this has a weird line ending, could you check it?

It seems fine in:
- https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/104

Reply:
![image](https://github.com/user-attachments/assets/7fd81a58-41a8-4ce4-9244-877be60b99dc)

Reply:
Thanks, fixed it!